frontend/app.tsx

```diff
@@ -29,6 +32,15 @@ export default function App({ Page, pageProps }: { Page: ComponentType<any>; pag
         <meta name="viewport" content="width=device-width" />
         <link rel="stylesheet" href="./style/site.css" />
         <link rel="stylesheet" href="./style/reset.css" />
+
+        <meta name="twitter:card" content="summary" />
+        <meta name="twitter:site" content="@hampus_s" />
```

Comment:
Could you make this as a config instead? You can add it to the `frontend` section of the config.
The prop should probably be optional.

Reply:
Added the creator from the config, is that just what you wanted or did you want all the options?